### PR TITLE
28 add tenant profil

### PR DESCRIPTION
**Pull request by bdtkevin**
Created routes for users (get / post / put)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +29,8 @@
     "cors": "^2.8.5",
     "dotenv": "^9.0.2",
     "express": "^4.17.1",
-    "mysql2": "^2.2.5"
+    "mysql2": "^2.2.5",
+    "nodemon": "^2.0.7"
```
Comment (ThomasSupervil) [inline package.json]: Nodemon should be in dev dependencies

**Review by ThomasSupervil — CHANGES_REQUESTED** [review]
minor change to do

**Review by ThomasSupervil — APPROVED** [review]
good

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**